### PR TITLE
When configuring with WITH_DOCS, but Doxygen is not available, prevent generation.

### PR DESCRIPTION
**Pull request by fidergo-stephane-gourichon**
…t generation.

### CODE REVIEW

File: doc/doxygen/CMakeLists.txt [hunk 1]
```diff
@@ -2,6 +2,8 @@ if(WITH_DOCS)
   find_package(Doxygen)
```
[the remark this answers is not in the document]
Comment (fidergo-stephane-gourichon) [reply doc/doxygen/CMakeLists.txt]: Seems better. I modify the code.

**Comment by SergioRAgostinho** [comment]
It's a Travis CI problem. Disregard it.

**Comment by SergioRAgostinho** [comment]
You seemed to have pushed a new commit after my comment, but nothing was modified.

**Comment by fidergo-stephane-gourichon** [comment]
Thanks for reporting wrong commit pushed.

I just pushed the correct commit, rebased on latest PointCloudLibrary/pcl master.

Tested here: 
* default configuration, without doxygen, cmake succeeds
* WITH_DOCS enable, without doxygen, cmake fails
* WITH_DOCS enable, with doxygen, cmake succeeds

Seems all good for me.

**Comment by SergioRAgostinho** [comment]
Thank you for contributing.

**Pull request merged**